flocker/provision/_install.py

```diff
@@ -114,6 +114,37 @@ def task_upgrade_kernel():
     ]
 
 
+KOJI_URL_TEMPLATE = (
+    'https://kojipkgs.fedoraproject.org/packages/kernel'
+    '/{version}/{release}.{distribution}/{architecture}'
+    '/kernel-{version}-{release}.{distribution}.{architecture}.rpm'
+)
+
+
+def koji_kernel_url(version, release, distribution, architecture):
+    """
+    Return the koji URL for the given kernel version.
+    """
+    url = KOJI_URL_TEMPLATE.format(
+        version=version,
+        release=release,
+        distribution=distribution,
+        architecture=architecture
+    )
+    return url
+
+
+def task_install_kernel(version='3.16.6', release='203', distribution='fc20',
```

Comment:
This seems to be called only from one place (and that one place specifies these arguments), and so doesn't need these defaults. In particular this is the case because the version specified here is not the one given by the one caller, and so it is never exercised.